flocker/control/httpapi.py

```diff
@@ -431,7 +431,8 @@ def state_datasets(self):
         examples=[
             u"create container",
             u"create container with duplicate name",
-            u"create container with ports"
+            u"create container with ports",
+            u"create container with environment"
```

Comment:
Put a trailing comma on this line **now** and the line won't show up as modified when you add an example for the next branch. :wink: :question: